priv/emqx.schema

```diff
@@ -2494,6 +2494,32 @@ end}.
   {datatype, {enum, [true, false]}}
 ]}.
 
+%% @doc Configuration of disconnected event reason.
+%% `normal`: normal disconnected events(by disocnnect mqtt packet)
+%% `kickout`: client was kicked out manually with management API.
+%% `takeover`: session was takenover by another client with same client ID. (clean_session = false)
+%% `discard`: session was takeover by another client with same client ID. (clean_session = true)
+{mapping, "broker.client_disconnect_normal", "emqx.client_disconnect_normal", [
+  {default, on},
+  {datatype, flag}
+]}.
+
+{mapping, "broker.client_disconnect_kicked", "emqx.client_disconnect_kicked", [
+  {default, on},
+  {datatype, flag}
+]}.
```

Comment:
I wonder if anyone will need to set them to `off`.
i.e. maybe no need to add them at all.